perceval/components/source.py

```diff
@@ -69,81 +71,65 @@ def __init__(self, brightness: float = 1,
         assert self._indistinguishability_model in ["homv", "linear"], "invalid value for indistinguishability_model"
         self._context = context or {}
         if "discernability_tag" not in self._context:
-            self._context["discernability_tag"] = 1
+            self._context["discernability_tag"] = 0
```

Comment:
Was it a bug ?
I don't see unit test modification.

Reply:
It is now incremented within the get_tag function, before being given instead of after, so it doesn't change anything